src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -378,6 +390,8 @@ h2o4gpu.elastic_net_classifier <- function(
   h2o4gpu_model(model, c("classifier"), "Elastic Net Classifier")
 }
 
+#' @description KMeans Clustering
```

Comment:
Is this title generated automatically?  If not, can we change to "K-means"?  (That's the official spelling).

Reply:
I'll change that.